### DISCUSSION TITLE
Cannot render local images

### DISCUSSION THREAD

**Discussion by davesaah** [Ice-Hazymoon/hugo-theme-luna#53] [Q&A] [answered]
I have been trying to render images for my post, but I keep getting nothing. 

![2022-12-27T21:16:47+00:00](https://user-images.githubusercontent.com/53381103/209723225-24dfd875-beea-45e9-8429-db0634bb767b.png)

This is the markdown in my file:

```md
---
title: "Test Post"
date: 2022-12-26T22:14:58Z
draft: true
slug: test
toc: true
categories:
    - "mentee or client"
summary: Testing how the post looks like.
tags:
    - intro
---

![an image](/static/images/logo.jpg)

```

**Comment by Ice-Hazymoon** [comment, marked as answer]
fixed on https://github.com/Ice-Hazymoon/hugo-theme-luna/commit/7fc94d775d6510e1d8831ccab2730b545dc791ba

try:
```markdown
![an image](/images/logo.jpg)
```

**Reply by davesaah** [reply]
It worked, thank you 🙂

**Reply by davesaah** [reply]
Very much appreciated
